service/gcs/bridge/bridge.go

```diff
@@ -126,6 +129,8 @@ func (mux *Mux) ServeMsg(r *Request) (RequestResponse, error) {
 
 // Request is the bridge request that has been sent.
 type Request struct {
+	// Context is the request context received from the bridge.
+	Context context.Context
```

Comment:
Do we send a `context.Context` over the wire? Does that work?

Reply:
We do not. I will extend the protocol to pass trace/span id in the json message but that will come when we have the shim support OC